CWE/events/trieste.txt

```diff
@@ -78,6 +80,50 @@ country_event = {
 		badboy = 5
 		relation = { who = USA value = -100 } 
 		relation = { who = ITA value = -50 }
+		USA = { country_event = 8020309 }
+		ai_chance = { factor = 0.1 }
+	}
+	
+}
+
+#Free Territory of Trieste
+country_event = {
+	id = 8020309
+	title = "The Yugoslavs don't respect the treaties"
+	desc = EVT_8020309_DESC
+	picture = "nwo2_free_city_of_trieste"
+  
+	is_triggered_only = yes 
+
+	option = {
+		name = "Return Zone A to Italy" 
+		inherit = TRE
+		736 = { secede_province = ITA } 
+		relation = { who = ITA value = 50 } 
+		relation = { who = YUG value = -50 } 
+		ai_chance = { factor = 0.8 }
+	}
+  
+	option = {
+		name = "Yugoslavia ought to respect the treaties" 
+		badboy = 5
+		relation = { who = YUG value = -100 } 
+		relation = { who = ITA value = 50 }
+		war = { 
+			target = YUG
+			attacker_goal = { casus_belli = free_allied_cores }
```

Comment:
Did you test whether free allied cores works for this event? Iirc this CB sometimes doesn’t work for scripted events.

Reply:
I haven't reached that part of testing yet as I didn't know that if I committed to my branch you would see it and be already part of the pull. As of now no event after the referendum seems to fire and when I try to console them it says they don't exist and I'm searching the cause, so I will check that later